fed/mosaic-application/src/test/java/org/eclipse/mosaic/fed/application/ambassador/simulation/electric/ChargingStationIndexTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2025 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.application.ambassador.simulation.electric;
+
+import static org.eclipse.mosaic.lib.geo.GeoPoint.latLon;
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.mosaic.fed.application.ambassador.simulation.electric.objects.ChargingStationObject;
+import org.eclipse.mosaic.fed.application.ambassador.simulation.electric.providers.ChargingStationIndex;
+import org.eclipse.mosaic.lib.geo.GeoCircle;
+import org.eclipse.mosaic.lib.geo.GeoPoint;
+import org.eclipse.mosaic.lib.junit.GeoProjectionRule;
+import org.eclipse.mosaic.lib.objects.electricity.ChargingSpot;
+import org.eclipse.mosaic.lib.objects.electricity.ChargingStationData;
+import org.eclipse.mosaic.lib.objects.electricity.ChargingType;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChargingStationIndexTest {
+    GeoPoint position = latLon(52.5, 13.4);
```

Comment:
Don't import `latLon` directly, do this instead:
```suggestion
    private final GeoPoint position = GeoPoint.latLon(52.5, 13.4);
```
Also adjust access modifiers